### PR TITLE
Edit multi-cloud docs

### PR DESCRIPTION
**Pull request by sophia-wiley**
Made some edits to multi-cloud docs.

### CODE REVIEW

File: docs/cloud/managing-airbyte-cloud.md [hunk 1]
```diff
@@ -85,22 +85,18 @@ To switch between workspaces:
...
+Default data residency allows you to choose where your data is processed. When you set the default data residency, it is applied to all new connections, but it does not affect existing connections or any currently running syncs. 
 
 For individual connections, you can choose a data residency that is different from the default. You can do this in the [connection settings](#choose-the-data-residency-for-a-connection) or when you create a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection).
 
-:::note 
-
-Your data is processed in the chosen residency, but some data is still stored in the US.
-
-:::
+Your data is processed on a data plane in the chosen residency, but some data, like data associated with the sync mode, cursor, and primary key, is still stored in the US on our control plane. For streams with data that must stay in the chosen data residency, do not configure sync modes that use cursor or primary key.
```
Comment (sophia-wiley) [inline docs/cloud/managing-airbyte-cloud.md:92]: @davinchia @andyjih Is this accurate? Do we want to include or remove any info?

Comment (sophia-wiley) [reply docs/cloud/managing-airbyte-cloud.md:92]: @malikdiarra

File: docs/cloud/getting-started-with-airbyte-cloud.md [hunk 2]
```diff
@@ -183,8 +183,7 @@ Verify the sync by checking the logs:
 3. Check the data at your destination. If you added a Destination Stream Prefix while setting up the connection, make sure to search for the stream name with the prefix.
 
 ## Allowlist IP address
-
-Depending on your data residency location, you may need to allowlist the following IP addresses to enable access to Airbyte:
+Depending on your [data residency](https://docs.airbyte.com/cloud/managing-airbyte-cloud#choose-your-default-data-residency) location, you may need to allowlist the following IP addresses to enable access to Airbyte:
 
 ### United States and Airbyte Default
 #### GCP region: us-west2
```
Comment (davinchia) [inline docs/cloud/getting-started-with-airbyte-cloud.md:189]: one correction: these should be us-west3

File: docs/cloud/getting-started-with-airbyte-cloud.md [hunk 3]
```diff
@@ -193,10 +192,12 @@ Depending on your data residency location, you may need to allowlist the followi
 * 34.106.60.246
 
 ### European Union
-#### AWS region: eu-west-3
```
Comment (davinchia) [inline docs/cloud/getting-started-with-airbyte-cloud.md:196]: eu-west-3 is correct for aws

File: docs/cloud/managing-airbyte-cloud.md (hunk 1 above)
Comment (davinchia) [inline docs/cloud/managing-airbyte-cloud.md:92]:
```suggestion
Although your data is processed on a data plane in the chosen residency, configuration data e.g. data associated with the sync mode, cursor, and primary key, is still stored on our control plane in the US. Data that must stay in the chosen data residency should not be used as a stream's cursor or primary key.
```

Comment (davinchia) [reply docs/cloud/managing-airbyte-cloud.md:92]: Some reword to clarify that the streams can use cursors/primary keys as long as the sensitive data is neither.

**Review by davinchia — APPROVED** [review]

File: docs/cloud/getting-started-with-airbyte-cloud.md (hunk 2 above)
Comment (davinchia) [reply docs/cloud/getting-started-with-airbyte-cloud.md:189]: AWS is in eu-west-3

**Pull request merged**